_code-samples/batch/js/multiAccountBatch.js

```diff
@@ -0,0 +1,134 @@
+console.log(`Bob: ${bob.address}, Balance: ${await client.getXrpBalance(bob.address)} XRP`)
+console.log(`Charlie: ${charlie.address}, Balance: ${await client.getXrpBalance(charlie.address)} XRP`)
+console.log(`Third-party wallet: ${thirdPartyWallet.address}, Balance: ${await client.getXrpBalance(thirdPartyWallet.address)} XRP`)
+
+// Create inner transactions  --------------------------------------------
+// REQUIRED: Inner transactions MUST have the tfInnerBatchTxn flag (0x40000000).
+// This marks them as part of a batch (allows Fee: 0 and empty SigningPubKey).
+
+// Transaction 1: Charlie pays Alice
+const charliePayment = {
+  TransactionType: "Payment",
+  Account: charlie.address,
+  Destination: alice.address,
+  Amount: xrpl.xrpToDrops(50),
+  Flags: xrpl.GlobalFlags.tfInnerBatchTxn // THIS IS REQUIRED
+}
+
+// Transaction 2: Bob pays Alice
+const bobPayment = {
+  TransactionType: "Payment",
+  Account: bob.address,
+  Destination: alice.address,
+  Amount: xrpl.xrpToDrops(50),
+  Flags: xrpl.GlobalFlags.tfInnerBatchTxn // THIS IS REQUIRED
+}
+
+// Send Batch transaction --------------------------------------------
+console.log("\n=== Creating Batch transaction... ===")
+const batchTx = {
+  TransactionType: "Batch",
+  Account: thirdPartyWallet.address,
+  Flags: xrpl.BatchFlags.tfAllOrNothing, // tfAllOrNothing: All inner transactions must succeed
+  // Must include a minimum of 2 transactions and a maximum of 8 transactions.
+  RawTransactions: [
+    { RawTransaction: charliePayment },
+    { RawTransaction: bobPayment },
+  ]
+}
+console.log(JSON.stringify(batchTx, null, 2))
+
+// Validate the transaction structure
+xrpl.validate(batchTx)
+
+// Set the expected number of signers for this transaction.
+// "autofill" will automatically add Fee: "0" and SigningPubKey: "".
+const autofilledBatchTx = await client.autofill(batchTx, 2)
+
+// Gather batch signatures --------------------------------
+// Each signer needs their own tx copy because signMultiBatch modifies the object.
+// Charlie signs the Batch transaction
+const charlieBatch = { ...autofilledBatchTx }
+xrpl.signMultiBatch(charlie, charlieBatch)
+
+// Bob signs the Batch transaction
+const bobBatch = { ...autofilledBatchTx }
+xrpl.signMultiBatch(bob, bobBatch)
+
+// Combine inner transaction signatures.
+// This returns a signed transaction blob (hex string) ready for submission.
+const combinedSignedTx = xrpl.combineBatchSigners([charlieBatch, bobBatch])
+
+// Submit the signed blob with the third-party's wallet
+console.log("\n=== Submitting Batch transaction... ===")
+const submitResponse = await client.submitAndWait(combinedSignedTx, 
+  { wallet: thirdPartyWallet }
+)
+
+// Check Batch transaction result --------------------------------
+if (submitResponse.result.meta.TransactionResult !== "tesSUCCESS") {
+  const resultCode = submitResponse.result.meta.TransactionResult
+  console.warn(`\nTransaction failed with result code ${resultCode}`)
+  process.exit(1)
```

Comment:
```suggestion
  await client.disconnect()
  process.exit(1)
```
I think it'd be good to cleanly close out the connection as a best practice.